test/rules/abapdoc.ts

```diff
@@ -94,6 +94,13 @@ const defaultConfigTests = [
                   doobar RETURNING VALUE(rv_string) TYPE string.
             ENDCLASS.`, cnt: 0,
   },
+  {
+    abap: ` CLASS zcl_foo DEFINITION PUBLIC.
+              PUBLIC SECTION.
+                METHODS:
+                  foobar RETURNING VALUE(rv_string) TYPE string REDEFINITION,
```

Comment:
i dont think theres a signature when redefining a method (because its already known from the superclass), so it should be just
```abap
METHODS:
  foobar REDEFINITION.
```